CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -226,17 +250,30 @@ class SchedulingPlacesViewController: UIViewController {
 extension SchedulingPlacesViewController: UICollectionViewDelegate {
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        let section = locationSections[indexPath.section]
-        switch section {
-        case .campus(let locations):
-            selectedCampusLocations.append(locations[indexPath.row])
-        case .ctown(let locations):
-            selectedCtownLocations.append(locations[indexPath.row])
-        }
-        if let cell = collectionView.cellForItem(at: indexPath) as? SchedulingPlaceCollectionViewCell {
-            cell.changeSelection(selected: true)
+        // Only three locations can be selected
+        if selectedCampusLocations.count + selectedCtownLocations.count < 3 {
+            let section = locationSections[indexPath.section]
+            switch section {
+            case .campus(let locations):
+                let location = locations[indexPath.row]
```

Comment:
lets define this before the switch since we use it in multiple of the cases

Reply:
I couldn't do it before switch cuz can't access `locations` outside the switch. The locations are dependent on the case of section

Reply:
ah gotcha